packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -425,6 +427,9 @@ const DataGridProVirtualScroller = React.forwardRef<
         <GridVirtualScrollerRenderZone {...getRenderZoneProps()}>
           {mainRows}
         </GridVirtualScrollerRenderZone>
+        <GridVirtualScrollerRenderZone {...getFocusRenderZoneProps()}>
+          {getFocusedRow()}
```

Comment:
Moving focus to checkbox cell acts weird when focus is out of view:


https://user-images.githubusercontent.com/13808724/192770636-46bed64b-534a-42f5-97e1-87e7ef455e10.mov